russian/wca-regulations.md

```diff
@@ -203,30 +205,30 @@
         - 7f1c) Таймер: таймер стакмата или секундомер (для долгих попыток).
         - 7f1d) Поверхность: плоская поверхность, на которой располагается стакмат. Мат считается частью поверхности. Таймер не считается частью поверхности.
     - 7f2) Таймер должен быть прикреплен к мату и расположен на поверхности, располагаясь на ближней к участнику стороне мата.
-        - 7f2a) Исключение: для сборки 3x3x3 ногами стакмат с матом должен быть расположен прямо на полу. Таймер может быть расположен на дальней от участника стороне мата.
+    - 7f3) Команде организаторов следует модифицировать стакмат, чтобы избежать случайного нажатия кнопок (например, использовать кольца вокруг кнопок), а также защитить батарейки (например, заполнив батарейный отсек).
```

Comment:
I don't like the wording here. "To secure" here doesn't have the same meaning as "protect", it's rather "fix in place", and "padding the battery compartment" isn't the same as just "fill the battery compartment" (that's too vague). 
A pad is something soft to fill between the battery compartment and the battery itself. I suggest to reflect this in the translation with some concrete examples of what things you could use as padding:
```suggestion
    - 7f3) Команде организаторов следует модифицировать стакмат, чтобы избежать случайного нажатия кнопок (например, использовать резиновые кольца вокруг кнопок), а также более плотно прижать батарейку в отсеке (например, добавлением между батарейкой и крышкой кусочка бумаги или поролона).
```